packet_packer.go

```diff
@@ -78,10 +78,20 @@ func (p *packetPacker) PackPacket(stopWaitingFrame *frames.StopWaitingFrame, con
 func (p *packetPacker) packPacket(stopWaitingFrame *frames.StopWaitingFrame, leastUnacked protocol.PacketNumber, packetToRetransmit *ackhandler.Packet) (*packedPacket, error) {
 	// packetToRetransmit is only set for handshake retransmissions
 	isHandshakeRetransmission := (packetToRetransmit != nil)
```

Comment:
I think I made this comment previously: This would be much clearer with s/packetToRetransmit/handshakePacketToRetransmit/g ;)